packages/client/package.json

```diff
@@ -11,7 +11,7 @@
     "start": "yarn run serve",
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "test": "vue-cli-service test:unit --passWithNoTests",
```

Comment:
This argument belongs to Jest, not Mocha, and seems to be an vestige of a time when GOST used Jest (other references to such removed in #305). In Mocha, for some reason it prevented ARPA tests from running -- I think it got interpreted as a path that didn't match anything?